File: cfastlogging/examples/default.c
#include <stdlib.h>
#include <stdio.h>
#include "h/cfastlogging.h"
#include <string.h>

// File: default.c
//
// Sample library usage.
int main(void)
{
    Logging logging = logging_new_default();
    logging_trace(logging, "Trace Message");
    logging_debug(logging, "Debug Message");
    logging_info(logging, "Info Message");
    logging_success(logging, "Success Message");
    logging_warning(logging, "Warning Message");
    logging_error(logging, "Error Message");
    logging_fatal(logging, "Fatal Message");
    logging_shutdown(logging, 0);
    return 0;
}

File: fastlogging/examples/default.rs
use fastlogging::{Logging, LoggingError};

fn main() -> Result<(), LoggingError> {
    let mut logger = Logging::default();
    logger.info("Info Message")?;
    logger.debug("Debug Message")?;
    logger.error("Error Message")?;
    logger.shutdown(false)?;
    Ok(())
}
